sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -89,6 +88,45 @@ describe("LocalWorkspace", () => {
         await Stack.createOrSelect(stackName, ws);
         await ws.removeStack(stackName);
     });
+    describe("Tag methods: get/set/remove/list", () => {
```

Comment:
Open to naming here. This seems to cover it, though - it's how I tested things locally. 😅